cli.js

```diff
@@ -35,6 +37,7 @@ const cli = meow(`
     --json         Output results as JSON
     --mobile       Emulates a Nexus 5X (default=true)
     --load-page    Loads the page (default=true)
+    --output       Outputs an HTML report to the specified path
```

Comment:
I think we might want something like specifying the reporter, e.g. `istanbul report`, rather than individual options for each report type (JSON, pretty printed JSON, HTML, etc)